F/FastTransforms/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+using BinaryBuilder
+
+# Collection of sources required to build FastTransforms
+name = "FastTransforms"
+version = v"0.2.9"
+sources = [
+    "https://github.com/MikaelSlevinsky/FastTransforms/archive/v$(version).tar.gz" =>
+    "555e8fb19ad76ee888fbab7bfaf4c269416c1c5950e9428cbe08edb44c22bf35",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+if [[ ${target} != *darwin* ]]; then
+    if [[ ${target} == *mingw* ]]; then
+        cd $WORKSPACE/destdir/bin
+    else
+        cd $WORKSPACE/destdir/lib
+    fi
+    if [[ ${nbits} == 32 ]]; then
+        ln -sf libopenblas.${dlext} libblas.${dlext}
+    else
+        ln -sf libopenblas64_.${dlext} libblas.${dlext}
```

Comment:
Julia comes with the library called `liboopenblas64_`, not `libblas`, I you should really be linking against it rather than forcing a name of a library that may or may not be there.  This is what Arpack does, whose model you said you followed.

Also, note that in the BinaryBuilder environment there is the environment variable `${libdir}` which points to `${WORKSPACE}/destdir/bin` on Windows and `${WORKSPACE}/destdir/lib` elsewhere

Reply:
Arpack’s most recent build script uses OpenBLAS_jll: https://github.com/JuliaPackaging/Yggdrasil/blob/master/A/Arpack/build_tarballs.jl#L92

How is making a symlink to libblas.${dlext} (only to help the make file, which expects it) the issue?

Reply:
I'm referring to https://github.com/JuliaPackaging/Yggdrasil/blob/74c30cace25e4975a731c120e6dd84a0e6b5f785/A/Arpack/build_tarballs.jl#L60-L64, not to the name of the JLL package to use as dependency.

Reply:
So why do they have it as a dep?

Reply:
Ah maybe for the cblas.h header?

Reply:
I'm not sure what are you asking.

The dependency physically provides the binary file, but then you have to link to the library.  What you're doing is to change the name of the linked library by creating a symlink.  Users won't be able to find that library, unless they have another OpenBLAS installed on their computer in addition to the Julia one.

Reply:
I just pushed a change that defines `LIBOPENBLAS` rather than makes symlinks. But it still failed locally with the same error as above (a bunch of undefined references).

Reply:
> But it still failed locally with the same error as above (a bunch of undefined references).

Sure, the change I suggested has nothing to do with building, jut with runtime linking.